lib/elastic_apm/config.rb

```diff
@@ -69,6 +69,9 @@ class Config
     option :transaction_max_spans,             type: :int,    default: 500
     option :transaction_sample_rate,           type: :float,  default: 1.0
     option :verify_server_cert,                type: :bool,   default: true
+
+    option :use_experimental_sql_parsing,      type: :bool,   default: false
```

Comment:
Will there be a point when this is not experimental but we still have an option to use it? If so, we should probably not have the word `experimental` in the name.
If the plan is to just switch over to this implementation in a minor version and remove the feature flag, the inclusion of the word `experimental` isn't a problem.

Reply:
My proposal was the latter. This is the new default, soft-launched for eager users.

Reply:
sounds good